airflow/www/package.json

```diff
@@ -141,7 +141,8 @@
     "reactflow": "^11.7.4",
     "redoc": "^2.0.0-rc.72",
     "remark-gfm": "^3.0.1",
-    "swagger-ui-dist": "4.1.3",
+    "sanitize-html": "^2.12.1",
```

Comment:
Are you sure this dependency is needed? It was just removed when adding color logs. Or is this a merge-mistake? Can not see it used in JS code... (or I have tomatoes on my eyes?)

Reply:
Thanks. Yes, that's likely a bad merge.

Reply:
Removed.